Resolver/WidgetLayoutContentResolver.php

```diff
@@ -25,7 +25,7 @@ public function getWidgetStaticContent(Widget $widget)
             $layouts[$size] = @$this->layoutConfig[$parameters['layout'.ucfirst($size)]] ?: null;
         }
         $colMax = array_reduce($layouts, function ($carry, $item) {
-            return max(count($item), $carry);
+            return (null === $item) ? 0 : max(count($item), $carry);
```

Comment:
`max()` is used to return the greatest value in `count($item)` and `$carry`.

If `$item` is null, this function will always return `0`, even if `$carry` is greater than 0.